quickwit/quickwit-search/src/leaf.rs

```diff
@@ -549,7 +549,8 @@ pub async fn leaf_search(
 
     // In the future this should become `request.aggregation_request.is_some() ||
     // request.exact_count == true`
-    let run_all_splits = true;
+    let run_all_splits = request.aggregation_request.is_some()
+        || request.count_hits == i32::from(CountHits::CountAll);
```

Comment:
I think prost creates an accessor that gives you the enum. Something like `get_count_hits()` or something

Reply:
`count_hits()`